File: Master/Year 1/Semester 1/AMSS/Proiect/app/src/main/java/ro/robertto/rescuepets/presentation/activities/StartupActivity.java
package ro.robertto.rescuepets.presentation.activities;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.view.WindowManager;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import ro.robertto.rescuepets.R;

public class StartupActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        getSupportActionBar().hide();

        getWindow().setFlags(
                WindowManager.LayoutParams.FLAG_FULLSCREEN,
                WindowManager.LayoutParams.FLAG_FULLSCREEN
        );

        setContentView(R.layout.startup);

        new Handler().postDelayed(() -> {
            startActivity(new Intent(StartupActivity.this, LogInActivity.class));
            finish();
        }, 4000);
    }
}

File: Bachelor/Anul 3/Semestrul 2/Android/Proiect/EmergencyApp/app/src/main/java/com/lifeSavers/emergencyapp/activities/StartupActivity.kt
package com.lifeSavers.emergencyapp.activities

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import android.view.WindowManager
import androidx.appcompat.app.AppCompatActivity
import com.lifeSavers.emergencyapp.R

class StartupActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {

        supportActionBar?.hide()

        window.setFlags(
            WindowManager.LayoutParams.FLAG_FULLSCREEN,
            WindowManager.LayoutParams.FLAG_FULLSCREEN
        )

        super.onCreate(savedInstanceState)
        setContentView(R.layout.startup)

        Handler().postDelayed({
            startActivity(Intent(this@StartupActivity, LogInActivity::class.java))
            finish()
        }, 4000)
    }
}
